cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -255,6 +257,23 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_relationship_index() {
+    profileQuery(
+      title = "A simple relationship index example",
+      text = "In this example, the query will use a `KNOWS(since)` relationship index, if it exists. ",
+      queryText = "MATCH ()-[relationship:KNOWS { since: 1992 } ]->() RETURN relationship",
```

Comment:
I suspect that real life queries would often return the nodes instead of or in addition to the relationship. Maybe

```suggestion
      queryText = "MATCH (person)-[relationship:KNOWS { since: 1992 } ]->(friend) RETURN person, friend",
```

Reply:
That's a very good point.